docs/guides/qiskit-transpiler-service.mdx

```diff
@@ -94,6 +94,16 @@ The following examples demonstrate how to transpile circuits using the Qiskit Tr
     transpiled_circuit = cloud_transpiler_service.run(circuit)
     ```
 
+## Limits of the Qiskit Transpiler Service
+
+The service has some limits enabled, here we list the most relevant ones:
+
+- The Qiskit Transpiler Service allows to transpile up to 1 million two-qubit gates in either `ai` mode `ai="true"`, `ai="false"` or `ai="auto"`.
+- The maximum time allowed to run a transpilation process is 30 mins.
+- The maximum time to retrieve a transpilation result from the service after the transpilation process ends is 20 mins. After 20 mins your job will be removed.
```

Comment:
I think the results are removed after `30min` right now.

Reply:
I double-checked and in the service config I see 1200 for polling max time for results.